### PR TITLE
Update the documentation to install the hooks

### PR DESCRIPTION
**Pull request by FlorentClarret**
### What does this PR do?

Update the documentation to install the hooks

### Motivation

That's outdated.

### CODE REVIEW

**Comment by FlorentClarret** [comment]
* **#26485** <a href="https://app.graphite.dev/github/pr/DataDog/datadog-agent/26485?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @FlorentClarret and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by pducolin — APPROVED** [review]

File: docs/dev/agent_dev_env.md [hunk 1]
```diff
@@ -62,6 +62,35 @@ Below a live demo of how the tool works:
 
 ![deva_install](../public/assets/images/deva.gif)
 
+We also provide an invoke task to perform some sanity checks and enable some tools. To use it you can run:
```
Comment (rtrieu) [inline docs/dev/agent_dev_env.md:65]:
```suggestion
Datadog also provides an invoke task to perform some sanity checks and enable some tools. To use it, run:
```

File: docs/dev/agent_dev_env.md [hunk 2]
```diff
@@ -204,12 +233,7 @@ It is optional but recommended to install `pre-commit` to run a number of checks
 
 #### Installation
 
-To install it, run:
-
-```sh
-python3 -m pip install pre-commit
-pre-commit install
-```
+To install pre-commit, follow [these instructions](https://pre-commit.com/#installation). The `deva setup` will automatically enable the hooks but you can do it manually running: `pre-commit install`.
```
Comment (rtrieu) [inline docs/dev/agent_dev_env.md:236]:
```suggestion
To install pre-commit, follow [these instructions](https://pre-commit.com/#installation). The `deva setup` automatically enables the hooks, but you can do it manually running: `pre-commit install`.
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
Minor feedback to adhere to docs style guide, but otherwise looks good!

**Review by rtrieu — APPROVED** [review]
Looks good!

**Pull request merged**